alerting/src/main/kotlin/org/opensearch/alerting/settings/AlertingSettings.kt

```diff
@@ -25,6 +25,12 @@ class AlertingSettings {
             Setting.Property.NodeScope, Setting.Property.Dynamic
         )
 
+        val ALERTING_TRIGGER_MAX_ACTIONS = Setting.intSetting(
+            "plugins.alerting.trigger.max_actions",
+            LegacyOpenDistroAlertingSettings.ALERTING_TRIGGER_MAX_ACTIONS,
```

Comment:
Any new settings can have their default constants defined here in `AlertingSettings` instead of legacy since that was for existing settings when we migrated to OpenSearch. You can refer to `MAX_ACTIONABLE_ALERT_COUNT` below as an example.

Reply:
> Any new settings can have their default constants defined here in `AlertingSettings` instead of legacy since that was for existing settings when we migrated to OpenSearch. You can refer to `MAX_ACTIONABLE_ALERT_COUNT` below as an example.

Got it! So the `legacy` was something like old version before migated to Opensearch? Can i ask where did it get migrated here? is that from Elastic Search?